modules/workspace.nix

```diff
@@ -32,13 +32,34 @@ let
     };
   };
 
+  cursorType = with lib.types; submodule {
+    options = {
+      cursorTheme = lib.mkOption {
```

Comment:
Why is this not called `theme` instead?
So the option path looks like `workspace.cursor.theme` similar to `workspace.cursor.size` 🤔